analyzers/SonarAnalyzer.sln

```diff
@@ -57,7 +57,9 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "SonarAnalyzer.Shared", "src
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "GitHub", "..\.github\GitHub.shproj", "{532AC70E-90CB-48AD-86F4-16767C22314D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "IntegrationTests", "its\IntegrationTests.shproj", "{AF787E26-7AB1-4B08-AF05-07CB0ECDE4E5}"
```

Comment:
`ITs` is actually a real folder in the project structure, just as `Tests`.

Let's make it look the same as `Tests` folder in the project solution - beside the `Common Solution Items`.

Reply:
My original intention was to have structure like this
```
Solution `SonarAnalyzer`/ITs/<something>`
Solution `SonarAnalyzer`/Tests/SonarAnalyzer.UnitTest`
```
where I don't have a good idea for `<something>` (`ITs/ITs`, `ITs/Root`, `ITs/Files`, `ITs/Common`) to avoid `Common Solution Items` in the way
```
Solution `SonarAnalyzer`/Common Solution Items/its`
```